plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala

```diff
@@ -135,11 +135,7 @@ object SrcGenPlugin extends AutoPlugin {
     muSrcGenMarshallerImports := {
       muSrcGenSerializationType.value match {
         case SerializationType.Avro | SerializationType.AvroWithSchema =>
-          val bigDecimal = muSrcGenBigDecimal.value match {
-            case ScalaBigDecimalGen       => BigDecimalAvroMarshallers
-            case ScalaBigDecimalTaggedGen => BigDecimalTaggedAvroMarshallers
-          }
-          List(bigDecimal, JavaTimeDateAvroMarshallers)
```

Comment:
I don't fully understand why we don't need the imports for big decimal marshallers

Reply:
Good question. I'm not sure why I made this particular change - I think it's a mistake. But I'm planning to come back to the custom marshallers in a later PR.

I'll merge this as-is for now and revert this change in my next PR.